include/z_prenmi.h

```diff
@@ -1,15 +1,15 @@
 #ifndef Z64PRENMI_H
 #define Z64PRENMI_H
 
-#include "global.h"
-
-void PreNMI_Destroy(GameState* thisx);
-void PreNMI_Init(GameState* thisx);
+#include "z64game.h"
 
 typedef struct {
     /* 0x00 */ GameState state;
     /* 0xA4 */ u32 timer;
     /* 0xA8 */ UNK_TYPE4 unkA8;
```

Comment:
```suggestion
    /* 0xA4 */ u32 timer;
    /* 0xA8 */ UNK_TYPE unkA8[0x4];
```
Unless it's used?

Reply:
It is actually used, I'll change it to `s32`